File: python/calculadora.py
# calculator/calculadora.py
class Calculadora:
  def adicao(self, a, b):
    return a + b


  def subtracao(self, a, b):
    return a - b

  def multiplicacao(self, a, b):
    return a * b


  def divisao(self, a, b):
    if b == 0:
        raise ValueError("Erro: Divisão por zero não é permitida")
    return a / b

File: unitarios/javascript/calculadora.js
// Criando a classe Calculadora
class Calculadora {

  // método para adição de dois números
  adicao(a, b) {
    return a + b;
  }


   // método para subtração
  subtracao(a, b) {
    return a - b;
  }

  // método para multiplicação
  multiplicacao(a, b) {
    return a * b;
  }

  // método para divisão
  divisao(a, b) {
    if (b === 0) {
      throw new Error('Erro:Divisão por zero não permitida');
    }
    return a / b;
  }
}




// Exportando a classe para ser usada nos testes
module.exports = { Calculadora };

